json-rpc/spec.json

```
@@ -196,6 +196,94 @@
 				"$ref": "#/components/contentDescriptors/GasPrice"
 			}
 		},
+		{
+			"name": "eth_feeHistory",
+			"summary": "Transaction fee history",
+			"description": "Returns transaction base fee and effective priority fee history for the requested/supported block range.",
+			"params": [
+				{
+					"name": "blockCount",
+					"description": "Number of blocks in the requested range (positive integer; less may be returned if requesting the entire specified range is not supported).",
+					"required": true,
+					"schema": {
+						"$ref": "#/components/schemas/Integer"
+					}
+				},
+				{
+					"name": "newestBlock",
+					"description": "Newest (highest number) block of the requested range (the range between headBlock-4 and headBlock are guaranteed to be supported while the pending block and older history are optional to support).",
+					"required": true,
+					"schema": {
+						"$ref": "#/components/schemas/BlockNumberOrTag"
+					}
+				},
+				{
+					"name": "rewardPercentiles",
+					"description": "A list of percentile values (0..100 floating point) to sample from each block's miner rewards in ascending order, weighted by gas used.",
```

```suggestion
					"description": "A list of percentile values to sample from each block's miner rewards in ascending order, weighted by gas used.",
```
Please use the `schema` field to communicate the type information.